third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -98,6 +98,11 @@ try {
 
 
 
+
```

Comment:
Nit: Why so much whitespace?

Reply:
Matter of more easily copy-and-pasting (the main chunk of) `gReplayScript` into a proper editor 😅